vips/vip-194/vip-194.ts

```diff
@@ -1,33 +1,17 @@
 import { ProposalType } from "../../src/types";
 import { makeProposal } from "../../src/utils";
 import commands192 from "../vip-192/commands";
-import commands193 from "../vip-193/commands";
-
-const PRIME_LIQUIDITY_PROVIDER = "0x103Af40c4C30A564A2158D7Db6c57a0802b9217A";
+import commands from "./commands";
 
 export const vip194 = () => {
   const meta = {
     version: "v2",
     title: "Prime Program Setup",
     description: ``,
     forDescription: "I agree that Venus Protocol should proceed with setting the prime program",
-    againstDescription:
-      "I do not think that Venus Protocol should proceed with setting the prime program",
-    abstainDescription:
-      "I am indifferent to whether Venus Protocol proceeds with setting the prime program",
+    againstDescription: "I do not think that Venus Protocol should proceed with setting the prime program",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds with setting the prime program",
   };
 
-  return makeProposal(
-    [
-      ...commands192,
-      ...commands193,
-      {
-        target: PRIME_LIQUIDITY_PROVIDER,
-        signature: "resumeFundsTransfer()",
```

Comment:
i do not get it why we have removed this command(resumeFundsTransfer()) as in this vip(vip-194) as we have paused funds transfer in vip-192, and i think it was expected to resumeFundsTransfer in this vip, am i not getting anything?